ReactCommon/react/renderer/mapbuffer/tests/MapBufferTest.cpp

```diff
@@ -8,6 +8,7 @@
 #include <memory>
 
 #include <gtest/gtest.h>
+#pragma pack(pop, 1)
```

Comment:
Maybe we can contain pack changes to `MapBuffer` headers?

Reply:
Will do!